docs/QUICK_REFERENCE.md

```
@@ -0,0 +1,142 @@
+uv run pytest tests/path/to/test.py -v
+
+# Specific test function
+uv run pytest tests/path/to/test.py::test_name -v
+
+# With output
+uv run pytest -v -s
+```
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)